spring-cloud-circuitbreaker-resilience4j/src/main/java/org/springframework/cloud/circuitbreaker/resilience4j/ReactiveResilience4JCircuitBreaker.java

```diff
@@ -27,20 +27,27 @@
 import io.github.resilience4j.timelimiter.TimeLimiter;
 import io.github.resilience4j.timelimiter.TimeLimiterConfig;
 import io.github.resilience4j.timelimiter.TimeLimiterRegistry;
+import io.vavr.Tuple;
```

Comment:
Unless required by a resilence4j api, we'd like to limit hard dependencies on transitive dependencies.